tests/bluetooth/tester/nrf5340_hci_rpmsg.conf

```diff
@@ -3,11 +3,5 @@ CONFIG_BT_MAX_CONN=2
 CONFIG_BT_BUF_EVT_RX_COUNT=16
 CONFIG_BT_BUF_EVT_RX_SIZE=255
 CONFIG_BT_BUF_CMD_TX_SIZE=255
-CONFIG_BT_BUF_ACL_TX_SIZE=251
 
-# L2CAP SDU/PDU TX MTU
-CONFIG_BT_L2CAP_TX_MTU=247
-
-# The minimum value for this is
-# L2AP MPS + L2CAP header (4)
-CONFIG_BT_BUF_ACL_RX_SIZE=255
+CONFIG_BT_BUF_ACL_RX_SIZE=100
```

Comment:
The `CONFIG_BT_BUF_ACL_RX_SIZE=100` for app core is already in `prj.conf`, so lets remove this:
```suggestion
```

and we could have an overlay for the net core image instead, e.g. `nrf5340_hci_rpmsg_cpunet.conf` with content:
```
# Apply this overlay at hci_rpmsg controller build
CONFIG_BT_CTLR_CONN_ISO_LOW_LATENCY_POLICY=y
CONFIG_BT_CTLR_DATA_LENGTH_MAX=100
CONFIG_BT_BUF_ACL_RX_SIZE=100
```

Reply:
Then we need to remove
```
CONFIG_BT_BUF_ACL_TX_SIZE=251
# L2CAP SDU/PDU TX MTU
CONFIG_BT_L2CAP_TX_MTU=247
# The minimum value for this is
# L2AP MPS + L2CAP header (4)
CONFIG_BT_BUF_ACL_RX_SIZE=255
```
also from https://github.com/zephyrproject-rtos/zephyr/blob/main/tests/bluetooth/tester/overlay-le-audio.conf#LL13-L18 to not break le audio tests.